.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -7,7 +7,7 @@ assignees: ''
 
 ---
 
-<!--- 
+<!---
```

Comment:
Please limit PRs to only relevant changes in the future.

Reply:
PRs or commits? I can split it to a separate commit, no problem

Reply:
A PR is a collection of commits.

Reply:
Yeah, I understand, I was just making sure you actually meant the PRs. It's just that I find weird creating a whole separate PR just to remove two trailing whitespaces in a single file, but if you feel more comfortable with that, then sure, it's your call.

Reply:
I'm not suggesting creating a separate PR to remove trailing whitespaces. Just leave them in.